ocaml/idl/datamodel.ml

```diff
@@ -4220,6 +4224,68 @@ let alert =
     ()
 *)
 
+(** network sriov **)
+module Network_sriov = struct
+  let lifecycle = [Published, rel_kolkata, ""]
+
+  let sriov_configuration_mode = Enum ("sriov_configuration_mode",
+    [
+      "sysfs", "Configure network sriov by sysfs, do not need reboot";
+      "modprobe", "Configure network sriov by modbrope, need reboot";
```

Comment:
minor typo here

Reply:
I am merging it, please @krizex send a PR to fix the typo later on

Reply:
Thanks @jonludlam